data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,11 @@
 [
   {
+    "id": "forging_standard",
+    "type": "requirement",
+    "//": "Forging of steel items (per steel chunk)",
+    "tools": [ [ [ "forge", 20 ], [ "char_forge", 20 ], [ "oxy_torch", 20 ] ] ]
```

Comment:
Why temporarily?
Oxyacetylene and batteries have different "energy rates". They don't need to consume equal numbers of charges.